Documentation/ApiOverview/Authentication/CSRFlikeRequestTokenHandling.rst

```diff
@@ -15,18 +15,18 @@ main scope is to ensure a user actually has visited a page, before submitting
 data to the webserver.
 
 This token can only be used for HTTP methods `POST`, `PUT` or `PATCH`, but
-for instance not for `GET` request.
+for instance not for a `GET` request.
 
 The :php:`\TYPO3\CMS\Core\Middleware\RequestTokenMiddleware` resolves
 request tokens and nonce values from a request and enhances responses with
 a nonce value in case the underlying application issues one. Both items are
 serialized as a `JSON Web Token (JWT)`_ hash signed with `HS256`. Request tokens
 use the provided nonce value during signing.
 
-Session cookie names involved for providing the nonce value:
+Session cookie names involved for providing the `nonce`_ value depending on the request:
 
-*   `typo3nonce_[hash]` in case request served with plain HTTP
```

Comment:
I would keep this extra verbosity for clarity

Reply:
It is clearer to me if there is only "http" mentioned.

Reply:
Alright, I think it doesn't hurt and if you feel this is clearer, I can accept this.